_posts/2019-07-04-cbo-introduction.md

```diff
@@ -258,21 +258,18 @@ filter condition or are returned from the join.
 
 # Conclusion
 
-Summing up, Presto’s Cost-Based Optimizer is conceptually a very simple thing.
+Summing up, Trino’s Cost-Based Optimizer is conceptually a very simple thing.
 Alternative query plans are considered, the best plan is chosen and executed.
 Details are not so simple, though. Fortunately, to use
-[Presto]({{site.url}}/), one doesn’t need to know all these details.
+[Trino]({{site.url}}/), one doesn’t need to know all these details.
 Of course, anyone with a technical inclination that like to wander in database
-internals is invited to study [the Presto code]({{site.github_repo_url}})!
+internals is invited to study [the Trino code]({{site.github_repo_url}})!
 
-Enabling Presto CBO is really simple:
+Enabling Trino CBO is really simple:
 
 - set `optimizer.join-reordering-strategy=AUTOMATIC` and
   `join-distribution-type=AUTOMATIC` in your `config.properties`,
 - [analyze]({{site.url}}/docs/current/sql/analyze.html) your tables,
 - no, there is no third step. That’s it!
 
-Take Presto CBO for a spin today and let us know what is *your* Presto
-experience!
```

Comment:
why removed?

Reply:
I agree.. why remove and not just also rename